spec/values/calculation/round/three_arguments.hrx

```diff
@@ -0,0 +1,241 @@
+$c: 10px;
+a {b: round(3.8px + 4px, $c)}
+
+<===> operation/two_args/simplify/output.css
+a {
+  b: 10px;
+}
+
+<===>
+================================================================================
+<===> operation/nearest/simplify/input.scss
+a {b: round(nearest, 115px + 2px, 21px + 4px)}
+
+<===> operation/nearest/simplify/output.css
+a {
+  b: 125px;
+}
+
+<===>
+================================================================================
+<===> operation/up/input.scss
+a {b: round(up, 101px, 25px)}
+
+<===> operation/up/output.css
+a {
+  b: 125px;
+}
+
+<===>
+================================================================================
+<===> operation/down/input.scss
+a {b: round(down, 122px, 25px)}
+
+<===> operation/down/output.css
+a {
+  b: 100px;
+}
+
+<===>
+================================================================================
+<===> operation/to-zero/positive/input.scss
+a {b: round(to-zero, 120px, 25px)}
+
+<===> operation/to-zero/positive/output.css
+a {
+  b: 100px;
+}
+
+<===>
+================================================================================
+<===> operation/to-zero/negative/input.scss
+a {b: round(to-zero, -120px, 25px)}
+
+<===> operation/to-zero/negative/output.css
+a {
+  b: -100px;
+}
+
+<===>
+================================================================================
+<===> infinity/input.scss
+@use "sass:math";
+a {b: round(up, infinity, infinity)}
+
+<===> infinity/output.css
+a {
+  b: calc(NaN);
+}
+
+
+<===>
+================================================================================
+<===> unknown_variable/first_arg/input.scss
+a {
+  e: round(var(--c), 8px, 9px);
+}
+<===> unknown_variable/first_arg/output.css
+a {
+  e: round(var(--c), 8px, 9px);
+}
+
+<===>
+================================================================================
+<===> unknown_variable/third_arg/input.scss
+a {
+  d: round(up, 8px, var(--c));
+}
+<===> unknown_variable/third_arg/output.css
+a {
+  d: round(up, 8px, var(--c));
+}
+
+<===>
+================================================================================
+<===> unknown_number/input.scss
+a {
+  e: round(up, number, 9px);
+}
+
+<===> unknown_number/output.css
+a {
+  e: round(up, number, 9px);
+}
+
+<===>
+================================================================================
+<===> interpolation/input.scss
+a {
+  e: round(#{"up"}, number, 9px);
+}
+
+<===> interpolation/output.css
+a {
+  e: round(up, number, 9px);
+}
+
+<===>
+================================================================================
+<===> math/input.scss
+a {
+  e: round(10px + 2px, 8px, 9px);
+}
+<===> math/error
+Error: 12px must be either nearest, up, down or to-zero.
+  ,
+2 |   e: round(10px + 2px, 8px, 9px);
+  |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 2:6  root stylesheet
+
+<===>
+================================================================================
+<===> error/syntax/strategy/number/input.scss
```

Comment:
This isn't a syntax error—syntax errors are thrown at parse time, before the stylesheet is even evaluated.